pkg/k8sutils/k8sutils.go

```diff
@@ -594,6 +668,22 @@ func DeletePersistentVolumeClaim(pvc *v1.PersistentVolumeClaim) error {
 	return client.PersistentVolumeClaims(pvc.Namespace).Delete(pvc.Name, &meta_v1.DeleteOptions{})
 }
 
+// GetPersistentVolumeClaimStatus returns the status of the given pvc
+func GetPersistentVolumeClaimStatus(pvc *v1.PersistentVolumeClaim) (*v1.PersistentVolumeClaimStatus, error) {
+	var err error
```

Comment:
we are not using this err variable.. let's remove it
we could remove it from GetStorageClassParams as well, if we use := on line 619